upcoming_changes/README.rst

```diff
@@ -0,0 +1,36 @@
+This directory contains "news fragments" which are short files that contain a small **ReST**-formatted
+text that will be added to the next ``CHANGELOG``.
+
+The ``CHANGELOG`` will be read by **users**, so this description should be aimed to hyperspy users
```

Comment:
```suggestion
The ``CHANGELOG`` will be read by **users**, so this description should be aimed to eXSpy users
```

Reply:
Only noticed that you already added this file in https://github.com/hyperspy/exspy/pull/22. Will rebase after #22 is merged.